arduino-hal/src/port/trinket_pro.rs

```diff
@@ -0,0 +1,116 @@
+pub use atmega_hal::port::mode;
+pub use atmega_hal::port::Pin;
+
+avr_hal_generic::renamed_pins! {
+    type Pin = Pin;
+
```

Comment:
Also related to the documentation update...

```suggestion

    /// Pins of the **Trinket Pro**.
    ///
    /// This struct is best initialized via the [`arduino_hal::pins!()`][pins] macro.
```